site/content/community/get-involved.md

```diff
@@ -1,7 +1,9 @@
 ---
-title: Community
-type: docs
-menu: docs
+title: Get Involved !
```

Comment:
Change to "Community" since this gets rendered as the top level `h1` element

Reply:
Do you want Get Involved as the nav link or Community ? I thought Community was better but open to other views ..

Reply:
Given there is now a community section, this ends up being community/community, which isn't very nice

Reply:
yeah this is tricky. I'd want "Get Involved" in the sidebar and url but the page title should include community, maybe "Get involved in the community". Any way to do that? If not I'm ok with keeping "Get Involved"

Reply:
Not the way this is currently structured - the menu is auto-generated from the page titles, as is the header. We can circle back round to this later on and revisit the layout ?

Reply:
sgtm